src/components/calculator.js

```diff
@@ -1,33 +1,55 @@
 import './Calculator.css';
+import { useState } from 'react';
 import Text from './calculatorText';
+import calculate from '../Logic/calculate';
 
-const CalculatorElement = () => (
-  <>
-    <Text />
-    <div className="grid-container">
-      <input className="input-number" value={0} />
-      <button type="button">AC</button>
-      <button type="button">+/-</button>
-      <button type="button">%</button>
-      <button type="button">/</button>
-      <button type="button">7</button>
-      <button type="button">8</button>
-      <button type="button">9</button>
-      <button type="button">X</button>
-      <button type="button">4</button>
-      <button type="button">5</button>
-      <button type="button">6</button>
-      <button type="button">-</button>
-      <button type="button">1</button>
-      <button type="button">2</button>
-      <button type="button">3</button>
-      <button type="button">+</button>
-      <button type="button" className="item-seventeen">0</button>
-      <button type="button" className="item-eighteen">.</button>
-      <button type="button">=</button>
-    </div>
+const CalculatorElement = () => {
+  const [state, setState] = useState({
+    total: null,
+    next: null,
+    operation: null,
+  });
 
-  </>
-);
+  const handleButtonClicks = (buttonValue) => {
+    const outPut = calculate(state, buttonValue);
+    setState(outPut);
+    // console.log(outPut);
```

Comment:
- [Required] I think it would be nice if you delete this piece of commented code from your code base, commented code will make your project look messy. If you need to review a previous version of your code, you can always use git history. Please remove this.